library/core/functions.general.php

```diff
@@ -2014,16 +2014,16 @@ function walkAllRecursive(&$input, $callback) {
             }
             foreach ($input as $key => &$val) {
                 if (is_array($val) || is_object($val)) {
-                    call_user_func_array($walker, [&$val, $callback, $key]);
```

Comment:
This is not a material change. However, calling a callable this way is more efficient.